File: src/main/java/com/pomodorobackend/pomodoro/controller/UserController.java
package com.pomodorobackend.pomodoro.controller;

import com.pomodorobackend.pomodoro.dto.UserDto;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.ArrayList;
import java.util.List;

@RestController
@RequestMapping("api/v1/users")
public class UserController {
    public ResponseEntity<List<String>> getUsers(){
        UserDto user = UserDto.builder()
                .username("Kevin")
                .password("pass")
                .build();
        List<String> users = new ArrayList<>();
        users.add(user.getUsername());
        return ResponseEntity.ok(users);

    }
}

File: src/main/kotlin/site/pomodoroapp/pomodoroApp/controller/UserController.kt
package site.pomodoroapp.pomodoroApp.controller

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController
import site.pomodoroapp.pomodoroApp.dto.UserDto

@RestController
@RequestMapping("api/v1/users")
class UserController {

    @GetMapping()
    fun getUsers(): ResponseEntity<List<String>>{
        var user: UserDto = UserDto("Kevin", "pass")
        var users = ArrayList<String>()
        users.add(user.username)
        //test
        return ResponseEntity.ok(users);
    }
}